lib/workers/repository/dependency-dashboard.ts

```diff
@@ -145,7 +146,8 @@ export async function ensureDependencyDashboard(
   }
   let issueBody = '';
   if (config.dependencyDashboardHeader?.length) {
-    issueBody += `${config.dependencyDashboardHeader}\n\n`;
+    issueBody +=
+      template.compile(config.dependencyDashboardHeader, config) + `\n\n`;
```

Comment:
I don't see a need for backticks here
```suggestion
      template.compile(config.dependencyDashboardHeader, config) + '\n\n';
```
or you should use them
```suggestion
      `${template.compile(config.dependencyDashboardHeader, config)}\n\n`;
```